.github/workflows/build-test.yaml

```diff
@@ -153,7 +156,9 @@ jobs:
       - name: Install Rust ${{ matrix.rust }}
         run: rustup default ${{ matrix.rust }}
       - name: Install upstream libseccomp
-        run: sudo ./scripts/install_libseccomp.sh
+        uses: ./.github/actions/setup
```

Comment:
Oh, thanks the fix.
This fix should be done for the main as well. so it would be nice to fix the main at first , then let's do cherry-pick.

Reply:
We don't have a msrv_compat job on main (anymore).

Reply:
Oh, sorry. You're right.